src/isAnnotatedForRemoval.js

```diff
@@ -0,0 +1,7 @@
+// @flow weak
+
+export default function (node) {
+  const comments = node.trailingComments || [];
+
+  return !!comments.find(({ value }) => value.trim() === 'remove-proptypes');
```

Comment:
Let's keep it, I couldn't find a better function: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/find.